src/js/preshuffle.s

```diff
@@ -922,6 +925,7 @@ UpdateStatusBarDisplays:
 StoreObjectExp:
   lda ObjectExp,y
   sta $61
+  sty $62
```

Comment:
Maybe add a comment about where/when this is used?

I'm a little worried about using $61 and $62 here since they're clobbered in a number of places - so it should be clear that these are used very soon - i.e. before anything is multiplied.